pshtt/pshtt.py

```diff
@@ -720,6 +722,11 @@ def is_hsts_preloaded(domain):
     return domain.domain in preload_list
 
 
+# Whether a domain's parent domain is in Chrome's HSTS preload list.
+def is_parent_hsts_preloaded(domain):
+    return parent_domain_for(domain.domain) in preload_list
```

Comment:
The only nit I have is here - I would call `is_hsts_preloaded()` on the result of `parent_domain_for(domain.domain)`, instead of also saying `in preload_list`, since that's an implementation detail (keeping a global list of domains cached up-front). 

That way, if we ever change the implementation of how we manage looking up preload status, we only have to change it in one place.